pkg/config/config.go

```diff
@@ -365,6 +365,7 @@ func initConfig(config Config) {
 	config.BindEnvAndSetDefault("external_metrics.aggregator", "avg")                    // aggregator used for the external metrics. Choose from [avg,sum,max,min]
 	config.BindEnvAndSetDefault("external_metrics_provider.bucket_size", 60*5)           // Window to query to get the metric from Datadog.
 	config.BindEnvAndSetDefault("external_metrics_provider.rollup", 30)                  // Bucket size to circumvent time aggregation side effects.
+	config.BindEnvAndSetDefault("kubernetes_event_collection_timeout", 100)              // value in milliseconds
```

Comment:
others duration are in second. maybe we should also use second for this parameter. WDYT?

Reply:
Not sure about that, I think it makes sense to stick to ms and keep it as an integer and avoid a discrepancy on that. @CharlyF any idea on that? Otherwise I would have preferred to go for an actual Duration and retrieve that directly instead.

Reply:
Ok maybe just support the golang `time.Duration` format: "100ms" or "0.1s"

Reply:
if it's documented, it's fine to leave in ms. We already have this for dogstatsd's timeout.